Docker Compose.md

```diff
@@ -0,0 +1,127 @@
+# Docker compose
+
+`docker-compose` is a tool built on top of docker and is used to start multiple service containers and connect them together.
+
+Compose is usually used to create an entire development environment, a typical docker-compose will start up a service for database, backend, frontend, HTTP proxy, etc., connect them all together in a private network and expose public ports on the host.
+
+`docker-compose` can be used for creating production environments, but other solutions are often better suited for that and docker-compose is usually used to simplify setting up the development environment.
```

Comment:
FROM
`docker-compose` can be used for creating production environments, but other solutions are often better suited for that and docker-compose is usually used to simplify setting up the development environment.
TO
`docker-compose` can be used to create production environments, but there are other solutions which are often better suited for that. Because of that, docker-compose is usually used to simplify setting up the development environment.